add bulk tag screenshot (#22895)
DataDog/documentation: content/en/service_management/service_level_objectives/_index.md

### SLO tags

> SLO Tags can be used for filtering on the [SLO status page][2], creating [SLO saved views][17], or grouping SLOs to view. Tags can be added to SLOs in the following ways:

Reviewer: ```suggestion
SLO tags can be used for filtering on the [SLO status page][2], creating [SLO saved views][17], or grouping SLOs to view. Tags can be added to SLOs in the following ways:
```

- When you create or edit an SLO, you can add tags

> - From the SLO list view, you can add/update tags in bulk using the *Edit Tags* and the *[Edit Teams][18]* dropdown options at the top of the SLO list:

Reviewer: ```suggestion
- From the SLO list view, you can add and update tags in bulk using the *Edit Tags* and the *[Edit Teams][18]* dropdown options at the top of the SLO list.
```

> {{< img src="service_management/service_level_objectives/slo_bulk_tag.png" alt="SLO bulk tags" >}}

Reviewer: ```suggestion
{{< img src="service_management/service_level_objectives/slo_bulk_tag.png" alt="SLO list page displays the Edit Tag dropdown for bulk tag editing" >}}
```
